test/parallel/test-stream-writable-aborted.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const { Writable } = require('stream');
+
+{
+  const writable = new Writable({
+    write() {
+    }
+  });
+  assert.strictEqual(writable.writableAborted, false);
+  writable.destroy();
+  assert.strictEqual(writable.writableAborted, true);
+}
+
+{
+  const writable = new Writable({
+    read() {
+    }
+  });
+  assert.strictEqual(writable.writableAborted, false);
+  writable.end();
+  writable.destroy();
+  assert.strictEqual(writable.writableAborted, true);
```

Comment:
This test also emits `'finish'`, should we check that `writable.writableAborted` is `false` after the `'finish'` event is emitted?

Reply:
That shouldn't emit finish... sounds like a bug